jujugui/static/gui/src/app/store/env/controller-api.js

```diff
@@ -841,6 +841,138 @@ YUI.add('juju-controller-api', function(Y) {
         storageEndpoint: result['storage-endpoint'] || '',
         regions: regions
       };
+    },
+
+    /**
+      Returns the tags of cloud credentials for a set of users.
+
+      @method getTagsForCloudCredentials
+      @param {Array} userCloudPairs A list of (user-tag, cloud-tag) pairs for
```

Comment:
This must be the only actual client implementation that makes use of the bulk API. nice.